src/diffusers/models/transformers/transformer_flux.py

```diff
@@ -373,7 +373,6 @@ def forward(
         )
         encoder_hidden_states = self.context_embedder(encoder_hidden_states)
 
-        print(f"{txt_ids.shape=}, {img_ids.shape=}")
```

Comment:
I was actually fixing it here https://github.com/huggingface/diffusers/pull/9057. But okay.

Reply:
Understood, we can ignore this then.